Primo.js

```diff
@@ -391,54 +298,8 @@ var testCases = [
 				"publisher": "Ambrogio : da Caponago",
 				"attachments": [],
 				"tags": [
-					"LEGGI;ITALIA - STORIA MEDIOEVALE"
-				],
-				"notes": [],
-				"seeAlso": []
-			}
-		]
-	},
-	{
-		"type": "web",
-		"url": "http://bcujas-catalogue.univ-paris1.fr/CUJAS_V1:LSCOP_ALL:33CUJAS_ALEPH000070200",
-		"items": [
-			{
-				"itemType": "book",
-				"title": "Test pattern for living",
-				"creators": [
-					{
-						"firstName": "Nicholas",
-						"lastName": "Johnson",
-						"creatorType": "author"
-					}
-				],
-				"date": "1972",
-				"callNumber": "203.206",
-				"language": "eng",
-				"libraryCatalog": "bcujas-catalogue.univ-paris1.fr",
-				"numPages": "xx+154",
-				"place": "Toronto New York",
-				"publisher": "Bantam Books",
-				"attachments": [],
-				"tags": [
-					{
-						"tag": "301.16/1/0973"
-					},
-					{
-						"tag": "Mass media"
-					},
-					{
-						"tag": "Mass media -- Social aspects -- United States"
-					},
-					{
-						"tag": "Social aspects"
-					},
-					{
-						"tag": "United States"
-					},
-					{
-						"tag": "United States -- Social conditions -- 1960-"
-					}
+					"LEGGI",
+					"ITALIA - STORIA MEDIOEVALE"
```

Comment:
When I run this test in the Translator Editor, I get "LEGGI;ITALIA - STORIA MEDIOEVALE". Can you verify that this still works?

Also, how are you updating these tests? I don't believe the Translator Editor, which is what you should be using, will generate a tags array in this format.

Reply:
yes, you right, it one string. fixed it.
What do how I am updating these tests? this is my goal for this PR.

Reply:
I'm asking what method you're using to update the tests. How are you running the tests, and how are you editing them?

Reply:
manually by the chrome extension and Zotero software.

Reply:
Meaning you're looking at the fields saved to Zotero and manually editing the tests in the .js file?

For translator development, you need to be doing this in [Scaffold](https://www.zotero.org/support/dev/translators/scaffold) (Tools → Developer → Translator Editor). We can't accept manually updated tests, because they won't match our automated tests, which we depend on to make sure translators are working.

If the site runs JS on load that's necessary for the translator to work, you can add `"defer": true` to the top level of the test and click Populate in the Testing pane to reload the tests. That allows the last three tests in Primo 2018 to run, for example, though they don't currently pass. (The first three primo-qa tests should be removed or replaced with publicly accessible URLs.)

Let us know if you need help working with Scaffold.

Reply:
@dstillman I used the Scaffold for all the tests. All of them passed successfully besides one Primo2018 test:
http://primo-prod.u-paris10.fr/primo-explore/fulldisplay?vid=UPON&docid=SCD_ALEPH000546633&context=L&search_scope=default_scope
It complained about wrong item type. I looked at the code and it should be 'thesis'.
Not sure why I got this error. Can you take a look at it?

Reply:
@dstillman - can you take a look please?

Reply:
@dstillman ping you about the "Detection returned wrong item type" error message.
Thanks.

Reply:
The test checks whether the imported item type conforms to detected item type and that's never the case for non-book items since the translator always detects book. You can either see if you can improve the detect function or remove the test. It's not a critical error.

Reply:
@adam3smith Couldn't find a solution to this error, so removed the problematic test.
The PR is ready for review.